cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+  import { getDictionaryId, getUser } from "../../../utils";
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+cy.visit(
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+);
+cy.url().should(
+"contain",
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.visit('/actions/');
+  cy.url().should(
+    "contain", '/actions/'
+  );
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByTitle("Add concepts").click();
```

Comment:
So the problem here is that you're asking it to find the button by the `title` property with the value of "Add concepts", but in the [source code](https://github.com/openmrs/openmrs-ocl-client/blob/c3c7d5246006bd0619ee70e947ae2d59a992d392/src/apps/dictionaries/pages/AddBulkConceptsPage.tsx#L107), we don't set the `title` property. Consequently, in the rendered document, the mark-up to produce the document looks like this:

```
<div class="jss100">
	<a class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary" tabindex="0" role="button" aria-disabled="false" href="/actions">
		  <span class="MuiButton-label">
			  Add concepts
		  </span>
		  <span class="MuiTouchRipple-root"></span>
	</a>
</div>
```

(I got this by right-clicking on the button and selecting "Inspect"). Instead, you'll want to use `findByText()` here since the "Add concepts" string is the text of the node we want to click on.

Reply:
@ibacher thanks for the review let me fix it right away.